frame/support/src/dispatch.rs

```diff
@@ -2294,7 +2294,7 @@ mod tests {
 		// default weight.
 		assert_eq!(
 			Call::<TraitImpl>::aux_0().get_dispatch_info(),
-			DispatchInfo { weight: 10_000, class: DispatchClass::Normal, pays_fee: true },
+			DispatchInfo { weight: 10_000_000, class: DispatchClass::Normal, pays_fee: true },
```

Comment:
I highly recommend removing this horrible idea that I had before (using default of SimpleDisoatchInfo as the _base weight_), I think it makes everything easier: 

- weights.rs exports a `const MINIMUM_WEIGHT: Weight = 10_000_000`;
- use it in such places.
- remove the `Default` implementation of `SimpleDispatchInfo`.
- use it anywhere that we rely on `SimpleDispatchInfo::default()`.

Reply:
done